docker/dockerfile.tf

```diff
@@ -77,3 +77,5 @@ RUN if [ "$INSTALL_DISTRIBUTED_EMBEDDINGS" == "true" ]; then \
         cd /distributed_embeddings && git checkout ${TFDE_VER} && git submodule update --init --recursive && \
         make pip_pkg && pip install artifacts/*.whl && make clean; \
     fi
+
+ENV FORCE_TF_AVAILABLE=true
```

Comment:
maybe worth adding a comment here providing some background on this - That the env var is for the `transformers` package to look up. And required because `importlib.metadata.version("tensorflow")` doesn't work (raises PackageNotFoundError) for our custom version of tensorflow that ships in the container

Reply:
Now that I mentioned this, I realised that checking if this also applies in the image where we're getting tensorflow from might be worthwhile. (`nvcr.io/nvidia/tensorflow:22.08-tf2-py3`). The metadata version import works there. And this is because the [dist-info directory](https://packaging.python.org/en/latest/specifications/recording-installed-packages/#the-dist-info-directory) for tensorflow is present in the dist-packages directory there.

An alternative (or additional change to this env var) could be to copy over the dist-info directory along with the package perhaps

Reply:
Opened a PR here to add dist-info https://github.com/NVIDIA-Merlin/Merlin/pull/704